okio-files/src/commonMain/kotlin/okio/Filesystem.kt

```diff
@@ -109,6 +109,8 @@ abstract class Filesystem {
   @Throws(IOException::class)
   abstract fun delete(path: Path)
 
+  abstract val separator: String
```

Comment:
Difficult to get the name here. There's a path separator (`:` or `;`) and a directory separator (`\` or `/`).

I was halfway considering calling the val `slash` ?

Reply:
I went with the same naming as Java that has `separator` and `pathSeparator`. The reasoning was that people coming from Java shouldn't be disoriented. Maybe that doesn't work too well for native/js but yea coming up with something that works everywhere is hard.

Reply:
Moved to `Path.directorySeparator` using a platform `expect/actual`.